### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Mmalumbo**

### CODE REVIEW

File: src/tests/migration.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { join } from 'path';
+import { Sequelize } from 'sequelize';
+import { DotenvParseOutput } from 'dotenv';
+import Worker = require('tortoise');
+
+import * as iconv from 'iconv-lite';
+iconv.encodingExists('foo');
```
Comment (liwoo) [inline src/tests/migration.integration.test.ts]: what's this doing?

File: src/tests/migration.integration.test.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+import { migration, migrationDataElements, modes, fakeDhis2Server } from './fixtures';
+import { loadConfig } from '../config';
+import { connectToDatabase } from '../datasource';
+import { createMigrationModel, createMigrationDataElementsModel } from '../models';
+import { createWorker, Message } from '../worker/helpers';
+import { migrate } from '../migrate/migrate';
+
+let path: string;
```
Comment (liwoo) [inline src/tests/migration.integration.test.ts]: these should be `const`

File: src/tests/migration.integration.test.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  });
+
+  afterEach(async () => {
+    await connection.close();
+    await worker.destroy();
+    await server.close();
+  });
+
+  it('should send data successfully to dhis2', async () => {
+    server = await fakeDhis2Server(3000, modes.succeed);
```
Comment (liwoo) [inline src/tests/migration.integration.test.ts]: where is this `server` being used?

File: src/tests/migration.integration.test.ts (hunk 1 above)
Comment (Mmalumbo) [reply src/tests/migration.integration.test.ts]: i thot i was doing integration tests not chunks.

File: src/tests/migration.integration.test.ts (hunk 2 above)
Comment (Mmalumbo) [reply src/tests/migration.integration.test.ts]: made it a const.

File: src/tests/migration.integration.test.ts (hunk 3 above)
Comment (Mmalumbo) [reply src/tests/migration.integration.test.ts]: in all the tests, it mocks a dhis2 server.

**Pull request merged**